OracleDatabase/SingleInstance/dockerfiles/23.4.0/Containerfile.free

```diff
@@ -22,70 +22,71 @@ FROM oraclelinux:8 as base
+    "issues"="https://github.com/oracle/docker-images/issues" \
+    "volume.data"="/opt/oracle/oradata" \
+    "volume.setup.location1"="/opt/oracle/scripts/setup" \
+    "volume.setup.location2"="/docker-entrypoint-initdb.d/setup" \
+    "volume.startup.location1"="/opt/oracle/scripts/startup" \
+    "volume.startup.location2"="/docker-entrypoint-initdb.d/startup" \
+    "port.listener"="1521"
 
-ARG INSTALL_FILE_1="https://download.oracle.com/otn-pub/otn_software/db-free/oracle-database-free-23c-1.0-1.el8.x86_64.rpm"
+ARG INSTALL_FILE_1="https://download.oracle.com/otn-pub/otn_software/db-free/oracle-database-free-23ai-1.0-1.el8.x86_64.rpm"
 
 # Environment variables required for this build (do NOT change)
 # -------------------------------------------------------------
 ENV ORACLE_BASE=/opt/oracle \
-    ORACLE_HOME=/opt/oracle/product/23c/dbhomeFree \
+    ORACLE_HOME=/opt/oracle/product/23ai/dbhomeFree \
     ORACLE_SID=FREE \
     INSTALL_FILE_1=$INSTALL_FILE_1 \
     RUN_FILE="runOracle.sh" \
     PWD_FILE="setPassword.sh" \
     CREATE_DB_FILE="createDB.sh" \
     USER_SCRIPTS_FILE="runUserScripts.sh" \
-    CONF_FILE="oracle-free-23c.conf" \
+    CONF_FILE="oracle-free-23ai.conf" \
     CHECK_SPACE_FILE="checkSpace.sh" \
     CHECK_DB_FILE="checkDBStatus.sh" \
     SETUP_LINUX_FILE="setupLinuxEnv.sh" \
+    DECRYPT_PWD_FILE="decryptPassword.sh" \
     CONFIG_TCPS_FILE="configTcps.sh" \
     INSTALL_DIR="$HOME/install" \
     ORACLE_DOCKER_INSTALL="true" \
-    CHECKPOINT_FILE_EXTN=".created" 
+    CHECKPOINT_FILE_EXTN=".created"
 
 # Use second ENV so that variable get substituted
 ENV PATH=$ORACLE_HOME/bin:$PATH
 
 # Copy binaries
 # -------------
-COPY $CHECK_SPACE_FILE $CONF_FILE $SETUP_LINUX_FILE $RUN_FILE $PWD_FILE $CHECK_DB_FILE $CREATE_DB_FILE $USER_SCRIPTS_FILE $CONFIG_TCPS_FILE $INSTALL_DIR/
+COPY $CHECK_SPACE_FILE $CONF_FILE $SETUP_LINUX_FILE $RUN_FILE $PWD_FILE $CHECK_DB_FILE $CREATE_DB_FILE $USER_SCRIPTS_FILE $CONFIG_TCPS_FILE $DECRYPT_PWD_FILE $INSTALL_DIR/
 
 WORKDIR $INSTALL_DIR
 
 # hadolint ignore=DL3003,DL3033,SC2035
-RUN mkdir -p "$ORACLE_BASE" && \
-    mv "$RUN_FILE" "$PWD_FILE" "$CHECK_DB_FILE" "$CREATE_DB_FILE" "$USER_SCRIPTS_FILE" "$CONFIG_TCPS_FILE" "$ORACLE_BASE"/ && \
-    chmod ug+x *.sh && \
-    sync && \
-    ./"$CHECK_SPACE_FILE" && \
-    ./"$SETUP_LINUX_FILE" && \
-    chown -R oracle:oinstall "$ORACLE_BASE" /home/oracle && \
-    sed -i -r 's/(^session\s+include\s+system-auth)/# \1/' /etc/pam.d/su && \
-    mv "$CONF_FILE" /etc/sysconfig/ && \
+RUN mkdir -p "$ORACLE_BASE" &&
+    mv "$RUN_FILE" "$PWD_FILE" "$CHECK_DB_FILE" "$CREATE_DB_FILE" "$USER_SCRIPTS_FILE" "$CONFIG_TCPS_FILE" "$DECRYPT_PWD_FILE" "$ORACLE_BASE"/ &&
+    chmod ug+x *.sh &&
+    sync &&
+    ./"$CHECK_SPACE_FILE" &&
+    ./"$SETUP_LINUX_FILE" &&
+    chown -R oracle:oinstall "$ORACLE_BASE" /home/oracle &&
+    sed -i -r 's/(^session\s+include\s+system-auth)/# \1/' /etc/pam.d/su &&
+    mv "$CONF_FILE" /etc/sysconfig/ &&
```

Comment:
```suggestion
RUN mkdir -p "$ORACLE_BASE" &&
    mv "$RUN_FILE" "$PWD_FILE" "$CHECK_DB_FILE" "$CREATE_DB_FILE" "$USER_SCRIPTS_FILE" "$CONFIG_TCPS_FILE" "$DECRYPT_PWD_FILE" "$ORACLE_BASE"/ && \
    chmod ug+x *.sh && \
    sync && \
    ./"$CHECK_SPACE_FILE" && \
    ./"$SETUP_LINUX_FILE" && \
    chown -R oracle:oinstall "$ORACLE_BASE" /home/oracle && \
    sed -i -r 's/(^session\s+include\s+system-auth)/# \1/' /etc/pam.d/su && \
    mv "$CONF_FILE" /etc/sysconfig/ && \
```

Reply:
It got removed by local shell script formatting extn. Fixed now. Thanks